src/PowerShellEditorServices/Services/PowerShellContext/Utilities/CommandHelpers.cs

```diff
@@ -96,7 +96,7 @@ internal static class CommandHelpers
                 .FirstOrDefault();
 
             // Only cache CmdletInfos since they're exposed in binaries they are likely to not change throughout the session.
-            if (commandInfo.CommandType == CommandTypes.Cmdlet)
+            if (commandInfo?.CommandType == CommandTypes.Cmdlet)
```

Comment:
It would have crashed here before this change if `commandInfo` was `null`.  Now that the code will get past here, do the callers of this method properly handle a `null` return value?

Reply:
Yes. They handle a null value. The change I added with the caching is what didn't handle a null value